Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -11,6 +11,7 @@ import Mathlib.Algebra.Order.Floor
 import Mathlib.Algebra.Order.ToIntervalMod
 import Mathlib.Topology.Instances.Real
 import Mathlib.Topology.Connected.PathConnected
+import Mathlib.Topology.IsLocalHomeomorph
```

Comment:
can you run `#minimize_imports` on this file? I think there is some redundancy here